src/packer/packer.rs

```diff
@@ -51,6 +51,26 @@ impl DensePacker {
         (self.width, self.height)
     }
 
+    /// Set new size for this packer
+    pub fn resize(&mut self, width: i32, height: i32) {
+        assert!(width >= self.width && height >= self.height);
+
+        self.width = width;
+        self.height = height;
+
+        // Add a new skyline to fill the gap
+        // The new skyline starts where the furthest one ends
+        let mut left = 0;
+        for skyline in self.skylines.iter() {
+            left = max(left, skyline.left);

```

Comment:
I don't remember how all this stuff works exactly but shouldn't it be `left = max(left, skyline.right());` instead?

Also, is the loop necessary? Skylines are probably already sorted and you can just take the last one if I remember correctly.

Reply:
You are correct, I have amended those changes.